src/components/kubectl/port-forward.ts

```diff
@@ -28,24 +29,25 @@ interface PodFromDocument {
     readonly namespace?: string;
 }
 
+
 type PortForwardFindPodsResult = PodFromDocument | FindPodsResult;
 
 function isFindResultFromDocument(obj: PortForwardFindPodsResult): obj is PodFromDocument {
     return (obj as PodFromDocument).fromOpenDocument;
 }
 
 /**
- * Implements port-forwarding to a target pod in the `default` namespace.
+ * Implements port-forwarding to a target pod in the current namespace
  * @param explorerNode The treeview explorer node, if invoked from
  * tree view.
  */
-export async function portForwardKubernetes (explorerNode?: any): Promise<void> {
+export async function portForwardKubernetes (kubectl: Kubectl, explorerNode?: any, ): Promise<void> {
```

Comment:
The trailing comma in the parameter list looks odd to me - is this intentional?

Reply:
definitely unintentional. will fix and merge.